cloudwash/entities/resources/ocps.py

```diff
@@ -29,10 +29,15 @@ def cleanup(self):
 
 class CleanAWSOcps(CleanOCPs):
     def list(self):
+        resources = []
         time_threshold = calculate_time_threshold(time_ref=settings.aws.criteria.ocps.sla)
 
-        query = " ".join([f"tag.key:{OCP_TAG_SUBSTR}*", f"region:{self.client.cleaning_region}"])
-        resources = self.client.list_resources(query=query)
+        ocp_prefix = list(settings.aws.criteria.ocps.ocp_prefix) or [""]
+        for prefix in ocp_prefix:
+            query = " ".join(
+                [f"tag.key:{OCP_TAG_SUBSTR}{prefix}*", f"region:{self.client.cleaning_region}"]
+            )
+            resources.extend(self.client.list_resources(query=query))
```

Comment:
The code will not execute if the prefix is not provided. Are we making the prefix mandatory for resource listing ?

if not, I would make the code applicable for both the conditions like when prefix is given and when not.

Reply:
Also if we go with the mandatory prefix method, you don't need resources var definition earlier and then extend it.

But no harm.

Reply:
Ok, if not mandatory, then please mention a comment above the settings value that the prefix is not mandatory or `wont be considered if not given` for listing and cleaning  the resources in OCP.